vue/components/ui/input-platforme/input-platforme.vue

```diff
@@ -1,22 +1,52 @@
 <template>
-    <input
-        type="text"
-        class="input-platforme"
-        v-bind:value="value"
-        v-bind:placeholder="placeholder"
-        ref="input"
-        v-on:input="$emit('update:value', $event.target.value)"
-        v-on:focus="focused()"
-        v-on:blur="blurred()"
-    />
+    <div class="input-platforme">
+        <input
+            type="text"
+            class="input"
+            v-bind:value="value"
+            v-bind:placeholder="placeholder"
+            v-bind:disabled="disabled"
+            ref="input"
+            v-on:input="$emit('update:value', $event.target.value)"
+            v-on:focus="focused()"
+            v-on:blur="blurred()"
+        />
+        <div class="text">
+            <div class="text-container" v-if="showError">
+                <button-icon-platforme

```

Comment:
Your intentions were good but in fact this is not a `button-icon` because it's not something that you can click on. Therefore, just use a `span` and make the CSS you need for it to work.